<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TakeSnapshotDlg</class>
 <widget class="QDialog" name="TakeSnapshotDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Take Snapshot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This feature lets you take a snapshot of the current nodes, jobs, qmgr, and pbs_server data.  NOTE: To view snapshot data, select &quot;Snapshot file&quot; in the &quot;Data source&quot; combobox on the main screen. To replay a series of snapshots, click &quot;Replay Snapshots&quot; button.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_ScheduleMultipleSnapshots">
     <property name="text">
      <string>Schedule multiple snapshots?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_TakeSnapshot">
     <property name="title">
      <string>Take single snapshot:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_SnapshotFrequency">
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_ScheduleSnapshots_Minutes">
          <property name="text">
           <string>Take snapshot every:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_ScheduleSnapshots_Hours">
          <property name="text">
           <string>Take snapshot every:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBox_EndSnapshots_Minutes_Iterations">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_ScheduleSnapshots_Minutes">
          <property name="text">
           <string>minutes, and ending after:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_ScheduleSnapshots_Hours">
          <property name="text">
           <string>hours, and ending after:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="spinBox_EndSnapshots_Hours_Iterations">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_ScheduleSnapshots_Minutes">
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_ScheduleSnapshots_Hours">
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_ScheduleSnapshots_Seconds">
          <property name="text">
           <string>Take snapshot every:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_ScheduleSnapshots_Seconds">
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_ScheduleSnapshots_Seconds">
          <property name="text">
           <string>seconds, and ending after:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_EndSnapshots_Minutes">
          <property name="text">
           <string>iterations (0 means don't end)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_EndSnapshots_Seconds_Iterations">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4" colspan="2">
         <widget class="QLabel" name="label_EndSnapshots_Seconds">
          <property name="text">
           <string>iterations (0 means don't end)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_EndSnapshots_Hours">
          <property name="text">
           <string>iterations (0 means don't end)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_SaveSnapshotFileAs">
        <property name="text">
         <string>Base snapshot filename (will have date/time appended):</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_SnapshotFilename">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBrowse">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_Comment">
        <property name="text">
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_Comment"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_SaveIndividualNodeData">
        <property name="text">
         <string>Save data for each node (uses momctl -d3)*</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_SaveIndividualJobData">
        <property name="text">
         <string>Save data for each job (uses qstat -f)*</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_RefreshLists">
        <property name="text">
         <string>Refresh all lists before taking snapshot</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>*can take considerable time on clusters with many nodes or jobs</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TakeSnapshotDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TakeSnapshotDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
